core/src/macosX64Main/kotlin/Uuid.kt

```diff
@@ -0,0 +1,18 @@
+package com.juul.kable
+
+import com.benasher44.uuid.Uuid
+import com.benasher44.uuid.uuidFrom
+import platform.CoreBluetooth.CBUUID
+import platform.Foundation.NSUUID
+
+internal fun Uuid.toNSUUID(): NSUUID = NSUUID(toString())
+internal fun NSUUID.toUuid(): Uuid = uuidFrom(UUIDString)
+
+internal fun Uuid.toCBUUID(): CBUUID = CBUUID.UUIDWithString(toString())
+internal fun CBUUID.toUuid(): Uuid {
+    val fromUuid = when (UUIDString.length) {
+        4 -> "0000$UUIDString-0000-1000-8000-00805F9B34FB"
+        else -> UUIDString
+    }
+    return uuidFrom(fromUuid)
+}
```

Comment:
A little confused what's happening here. Is CBUUID not guaranteed to be an actual UUID?

Reply:
Doh. I'll look into that. This may be a poor choice of copy/paste. UUIDs can have shorthand notation, which Javascript supports; I may have copied this from my sample project on Javascript. Or Apple platforms also supports the shorthand notation; I'd be surprised if they did though, so I'll look into it.

Good catch.